### PR TITLE
WIP: Update community info.

### PR DESCRIPTION
**Pull request by Ismael-VC**
Please let me know of any suggestion, thanks in advance!

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
...
 # Chat Rooms
 
+## IRC
+
 * Many Julia users and developers hang out in the [#julia IRC channel](http://webchat.freenode.net/?channels=julia) on Freenode.
+
+## Gitter
+
 * [julia](https://gitter.im/JuliaLang/julia) Gitter chat room is for discussions around the usage of Julia.
-* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
```
Comment (waldyrious) [inline community/index.md]: Why was this removed?

File: community/index.md [hunk 2]
```diff
@@ -151,9 +229,15 @@ South America
...
+* [Julia Tokyo](https://juliatokyo.connpass.com/)
+
+
 # Reddit
 
 [The Julia Language SubReddit](http://www.reddit.com/r/Julia/) is a collection of various blog posts and articles related to Julia.
 
-
-[julia-users]: https://groups.google.com/group/julia-users
+Don't see your community resourses listed here? Please submit a [pull request](https://github.com/JuliaLang/julialang.github.com/edit/master/community/index.md) so we can share them!
```
Comment (waldyrious) [inline community/index.md]: Typo: should be "resources".

File: community/index.md [hunk 3]
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
...
+* [JunoIDE](https://gitter.im/JunoLab/Juno)
+* [JuliaML](https://gitter.im/JuliaML/chat)
+* [JuliaGL](https://gitter.im/JuliaGL/meta)
+* [BioJulia](https://gitter.im/BioJulia/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [IntelLabs](https://gitter.im/IntelLabs/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [UCIDataScienceInitiative](https://gitter.im/UCIDataScienceInitiative/home)
+* [Julia-i18n](https://gitter.im/Julia-i18n/julia-i18n) Gitter chat room is for discussions around Julia internationalization and localization.
+* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
```
Comment (Ismael-VC) [inline community/index.md]: @waldyrious here is the line you are looking for, notice that also the JuliaLangPt cahnel is now listed below too! 👍

File: community/index.md [hunk 4]
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
...
+* [JuliaLangPt/julia](https://gitter.im/JuliaLangPt/julia) Gitter chat room is for discussions around the usage of Julia in the Portuguese language.
+
+### Packages
+
+* [Gadfly.jl](https://gitter.im/dcjones/Gadfly.jl)
+* [LightGraphs.jl](https://gitter.im/JuliaGraphs/LightGraphs.jl)
+* [GR.jl](https://gitter.im/jheinen/GR.jl)
+* [FixedSizeArrays.jl](https://gitter.im/SimonDanisch/FixedSizeArrays.jl)
+* [DSGE.jl](https://gitter.im/FRBNY-DSGE/DSGE.jl)
+* [Docile.jl](https://gitter.im/MichaelHatherly/Docile.jl)
```
Comment (MichaelHatherly) [inline community/index.md]: Docile and Lexicon are both pretty much deprecated on Julia 0.4+ so best to not include those. Rather include the gitter room for the JuliaDocs org https://gitter.im/juliadocs/users in the section above. Could you add the JuliaDocs org to the organisation section as well, thanks @Ismael-VC.

Comment (Ismael-VC) [inline community/index.md]: Sure thing! I didn't know about that juliadocs chat room.

Comment (MichaelHatherly) [inline community/index.md]: Hadn't put much effort into publicising it... perhaps I won't need to now :smile:

File: community/index.md [hunk 5]
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
...
+* [Mathematica.jl](https://gitter.im/MikeInnes/Mathematica.jl)
+* [Nanopore.jl](https://gitter.im/gawbul/Nanopore.jl)
```
Comment (bicycle1885) [inline community/index.md]: Nanopore.jl is an empty package and no updates for two years. So, I guess this is abandoned.

Comment (Ismael-VC) [inline community/index.md]: I've just been searching gitter rooms, haven't checked yet if all those packages are registered and active, but will do.

File: community/index.md [hunk 6]
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
...
+* [JuliaPraxis](https://gitter.im/JuliaPraxis) gitter chat rooms are for discussions around Julia best practices.
+* [JuliaQuantum](https://gitter.im/JuliaQuantum/home)
+* [JunoIDE](https://gitter.im/JunoLab/Juno)
+* [JuliaML](https://gitter.im/JuliaML/chat)
+* [JuliaGL](https://gitter.im/JuliaGL/meta)
+* [BioJulia](https://gitter.im/BioJulia/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [IntelLabs](https://gitter.im/IntelLabs/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [UCIDataScienceInitiative](https://gitter.im/UCIDataScienceInitiative/home)
```
Comment (ChrisRackauckas) [inline community/index.md]: You're missing [JuliaDiffEq](https://gitter.im/JuliaDiffEq/Lobby). Also, the data science initiative one doesn't need to be here: it's just for an old workshop.

File: community/index.md (hunk 5 above)
Comment (ChrisRackauckas) [inline community/index.md]: You're missing the ArbFloats chatroom. Also, there's a lot of "defunct" rooms here. Most of them have no posts, or no posts in the last year. Maybe any room that hasn't had a post in a year should be dropped? It's just a bad showing to have a bunch of empty rooms.

I don't know what "the rule" should be, but rooms with 0 posts in 2 years? I think it's clear they just clutter the list.

**Comment by oxinabox** [comment]
Shall we leave the groups listed in #445 for another PR?
(or until owners volunteer them)

File: community/index.md (hunk 6 above)
Comment (Ismael-VC) [inline community/index.md]: Got it! I had it there in case you planned to repeat the workshop.

**Comment by Ismael-VC** [comment]
> (or until owners volunteer them)

@oxinabox what do you mean by *owner volunteer them*?

I think it's ok to add them here either way, I was unaware of # 444 and # 445. It's great that you've already discussed organizing that list.

**Comment by oxinabox** [comment]
>@oxinabox what do you mean by until owners volunteer them?

As in, perhaps there is a reason the creators/maintainers of some of those groups have not raised an issue/a PR to have them added.
So maybe we should wait until they do?

File: community/index.md [hunk 7]
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
...
+* [Julia-i18n](https://gitter.im/Julia-i18n/julia-i18n) Gitter chat room is for discussions around Julia internationalization and localization.
+* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
+* [JuliaLangPt/julia](https://gitter.im/JuliaLangPt/julia) Gitter chat room is for discussions around the usage of Julia in the Portuguese language.
+
+### Packages
+
+* [Gadfly.jl](https://gitter.im/dcjones/Gadfly.jl)
+* [LightGraphs.jl](https://gitter.im/JuliaGraphs/LightGraphs.jl)
+* [GR.jl](https://gitter.im/jheinen/GR.jl)
+* [FixedSizeArrays.jl](https://gitter.im/SimonDanisch/FixedSizeArrays.jl)
```
Comment (KristofferC) [inline community/index.md]: AFAIU, FixedSizeArrays is on the deprecation path in favour of StaticArrays.jl so maybe no point in including it here. @SimonDanisch likely has a more informed opinion than me.

File: community/index.md (hunk 3 above)
Comment (waldyrious) [inline community/index.md]: Got it :) the way the diff is displayed (with the removed "QQ groups" heading in the middle of this section) threw me off.

File: community/index.md (hunk 7 above)
Comment (SimonDanisch) [inline community/index.md]: Yeah, I almost finished transferring all my packages to StaticArrays :) Just a few bugs left, but then the deprecation for FixedSizeArrays in 0.6 should be pretty final!

**Comment by SimonDanisch** [comment]
I'd also add `JuliaGeometry` - https://github.com/JuliaGeometry

**Comment by ChrisRackauckas** [comment]
Bumping this so it doesn't get lost.

**Comment by ViralBShah** [comment]
Still needs to address some of the comments here (at least spellings)

**Comment by ViralBShah** [comment]
Bump.

**Comment by JeffreySarnoff** [comment]
last paragraph: spell as "resources"

**Comment by StefanKarpinski** [comment]
Bump, this should get merged – @Ismael-VC, can you update?

**Comment by Ismael-VC** [comment]
Certainly, I'm on it right now.

**Comment by Ismael-VC** [comment]
@JeffreySarnoff

> "Old lists" is misleading (no longer in use?) maybe
> "Domain Resources" or "Collaborative Repositories"

 * I have used "Domain Resources" instead.

@SimonDanisch @KristofferC 

* removed FixedSizeArrays reference
* is there a StaticArrays.jl gitter room?
* added JuliaGeometry

@ChrisRackauckas 

* added JuliaDiffEq

@waldyrious 

* fixed typo

@MichaelHatherly 

* added JuliaDocs

@bicycle1885 

* removed Nanopore

@oxinabox 

* added missing orgs from #445 
* orgs listed based on #444 and #477

**Comment by Ismael-VC** [comment]
@ViralBShah @StefanKarpinski I've enhanced, updated and rebased, please let me know if there is anything else that should be fixed or added, thanks in advance and sorry for the wait.

**Comment by ViralBShah** [comment]
I think further reorganization will be needed since there is a lot on this page. But merging for now as we can update in future PRs.
